declaration-diagnosis-redesign/choosing-answer-strategy.qmd

```diff
@@ -196,15 +199,22 @@ diagnosis_9.2 <- get_rdddr_file_local("diagnosis_9.2")
 
 #### Uncertainty
 
-Classic hypothesis tests do not provide estimates of uncertainty regarding any associated estimate. They do however let you impose stricter or weaker conditions before rejecting an null hypothesis. If you wanted to be very conservative you could stipulate that you will only reject a hypothesis if the associated $p$  values is less than $0.0001$. In this case you are not likely to reject many null hypotheses close to your estimate but those that you do reject you may feel you reject with greater confidence. Not to belabor the point, it is not that the procedure tells you how sure you should be that the null is wrong, it is just that you figure out that the data is very unusual if the null were right.   
+For tests, uncertainty is expressed by describing the properties of a procedure in terms of error rates. A test is an answer strategy that returns a binary answer to a binary inquiry. The result of a test is an error if the empirical answer $a_d$ does not equal the truth $a_{m^*}$. Conventionally, a Type 1 error occurs when $a_d = 1$ but $a_{m^*} = 0$ and a Type 2 error occurs when $a_d = 0$ but $a_{m^*} = 1$. A perfect test (i.e., a test about about which we are fully certain) has Type 1 error rate of 0% and a Type 2 error rate of 0% as well. A test about which we are less certain might return $a_d  = 1$ 40% of the time when $a_{m^*} = 0$ (a Type I error rate of 40%) and might return $a_d = 1$ 90% of the time when $a_{m^*} = 1$ (a Type II error rate of 10%).
+
+The test reported by the `answer\_strategy` function is a null hypothesis significance test against the null hypothesis that the average age in this Italian village is equal to exactly zero. The test returns "yes" if we reject the null and "no" if we fail to reject it. If we use the standard significance threshold of $alpha = 0.05$ we fail to reject the null model because the `p.value` reported in the table is 0.12. It's a silly test, but silly tests like these are reported by default in many statistical software languages and in many scientific papers to boot. It's a silly test because we always knew the average age was not zero!
```

Comment:
there should not be a \ in answer_\strategy (in latex that's right to escape the _ character, but in code here it will literally render the \.

Reply:
Corrected!